packages/jest-resolve-dependencies/src/index.ts

```diff
@@ -58,8 +58,12 @@ class DependencyResolver {
           dependency,
           options,
         );
-      } catch (e) {
-        resolvedDependency = this._resolver.getMockModule(file, dependency);
+      } catch (e1) {
```

Comment:
Since optional catch binding is in ES2019, I think we should be able to omit `(e1)` and `(e2)` here.

Reply:
I've removed them.